src/components/confirm-icons.tsx

```diff
@@ -0,0 +1,72 @@
+import { Box, Button, Chip, IconButton, SvgIconTypeMap } from '@mui/material';
+import React, { useState } from 'react';
+import { useTranslator } from '../hooks';
+import CloseIcon from '@mui/icons-material/Close';
+import { OverridableComponent } from '@mui/material/OverridableComponent';
+
+export function ConfirmIcon(props: {
```

Comment:
The names of these components is confusing. `ConfirmIcon` is not an icon but really a button. I suggest the following renames:


```
ConfirmButton => ConfirmDialogButton
ConfirmIcon => ConfirmButton
```

Rename the files accordingly as well.